src/routes/components/components.json

```
@@ -2362,6 +2362,16 @@
 		"category": "User Interaction",
 		"stars": 30
 	},
+	{
+		"title": "svelte-bricks",
+		"url": "https://github.com/janosh/svelte-bricks",
+		"description": "Naive Svelte masonry component without column balancing (ragged columns at the bottom)",
+		"tags": ["components and libraries", "masonry layout", "media sites"],
+		"npm": "svelte-bricks",
+		"addedOn": "2022-09-11T10:00:00Z",
+		"category": "Design Pattern",
```

```suggestion
		"category": "Data Visualisation",
```
`Design Pattern` are more [**Software Design Pattern**](https://en.wikipedia.org/wiki/Software_design_pattern) than _"Graphical Design Pattern"_

---

Ah, good to know.